source/index.html.md.erb

```diff
@@ -325,10 +335,14 @@ Before you begin, you should already have an access token, which you used in the
 {
 	"id": 6141831453927300,                         // This is the Id of the entire sheet
 	"name": "My first sheet",
-	"columns": [{                                   // Each Column Object connects column title to Id and defines the column details
+    "columns": [{                                   // Each Column Object connects column
```

Comment:
"connectes column title to Id" --> "associates column Id with column title" ??

Reply:
Done.